ab-testing/scripts/validation/limitServerSide.ts

```diff
@@ -0,0 +1,14 @@
+import { ABTest } from '../../types.ts';
+
+// TODO: This is a placeholder, actual number TBD
+export const MAX_SERVER_SIDE_TESTS = 20;
```

Comment:
Maybe a note here on why we have a limit - i.e. it controls how much server side tests split the cache by?

Reply:
Yep, good shout, have updated this